### ISSUE TITLE
input_numeric() warning when setting a too high value ignores html language

### ISSUE THREAD

**Issue by arianmustafa** [rstudio/py-shiny#561]
Hi, 
both my OS and browser are in german but I would prefer to have my shiny app completely in english.
When I now set a maximum or minimum value for a numeric input and type a number larger or smaller than that, the warning I get is in german. I used `lang="en"` of the page_fluid function to set my document language to english but this does not seem to work.

`app_ui = ui.page_fluid(
    otherelements()
    ), lang='en'
)`

**Comment by wch** [comment]
Are you referring to the tooltip that shows up when you hover your mouse cursor over the `input_numeric`? (Note that the mouse cursor doesn't show in the screen shot below)

<img width="858" alt="image" src="https://github.com/rstudio/py-shiny/assets/86978/3cb2af63-c76a-476e-b02e-310c5e3ab5a0">


I believe that message is set by the browser for `<input type="number>`.

**Comment by arianmustafa** [comment]
Yes, this is exactly what I meant. I probably should have searched for the corresponding html tag first, but I am completely new to anything in the realm of web development. I just noticed something different though. While the upper and lower limit prohibit me from exceeding the set limits, I can still type in numbers outside of the range which are processed instantly without a check for validity. If I understood correctly I would probably have to implement the input in html instead of using ui.input_numeric() to implement a forced maximum, right?

Sorry for the unnecessary issue still!

**Comment by wch** [comment]
> While the upper and lower limit prohibit me from exceeding the set limits, I can still type in numbers outside of the range which are processed instantly without a check for validity. If I understood correctly I would probably have to implement the input in html instead of using ui.input_numeric() to implement a forced maximum, right?

The behavior of numeric inputs is browser dependent, and for most browsers (maybe all browsers now), the `min` and `max` only affect the limits when you click on the up and down arrows -- they still let you type in any number.

We are actually doing some experiments with a new number input, which you can [see in action here](https://shinylive.io/py/editor/#code=NobwRAdghgtgpmAXGKAHVA6VBPMAaMAYwHsIAXOcpMAMwCdiYACAZwAsBLCbJjmVYnTJMAgujxMAkhFQBXMiwkB5eXIUSAynBYsOpCXUoATOHQmyOAHQjW+Aoa07cS-UpWFQWrQtd8R6jI5cPHaCwmKoBsam5lY20OgA+hZMALxMFlhQAOZwiTQANhZGABTWTBXeGDSCuRhcaokQsjAARqZlYAAelvhMvQByLe10vDLyvRIAblBFcKkAjAAMEjBci6tQXakATEsAlHjllZmtdCWHxxWZxKryiRRdZIlTpq1QZHydZE+9lxD7PzWEw0VimV7nBryCS3MhqCQsbS6Uj7RBXJgAAVhanRGMMEBMdAwjzI6JBTB+ZAuaIglTpTEMZFkdFpNF6XUQTBAULIGC6FwAvr0gQlUGlROgSmhUMkOAjwaZARAwAK8OBoPBqIYAI4WQzwcgsYlPfBEUgUKjINhkOEsRAAentdCgAHcMNkOGQ2LJWrJEXQSOR3BgXPaXYQ2Pb2MEXAIIO57TAoFwALS+jgFIz2nD2owcFhkKNObCxtzkFNLDCVhYpnAAZhTEDcKag3AwLrYBRVAF0gA). It won't accept values that are outside of the range. The code is still very experimental and will change, so it would not be good to use it now -- I mention it so that you know that we hope to have a better solution soon, in the next couple of months.
